CHANGELOG.md

```diff
@@ -4,8 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/at-gmbh/personio-py/compare/v0.2.1...HEAD)
+## [Unreleased](https://github.com/at-gmbh/personio-py/compare/v0.2.2...HEAD)
 
+
+## [0.2.2](https://github.com/at-gmbh/personio-py/tree/v0.2.2) - 2022-07-04
+
+* add new fields: 'updated_at' and 'category' to the Absence and AbsenceType classes
```

Comment:
can you add a link to this PR?

`([#26](https://github.com/at-gmbh/personio-py/pull/26))`